src/lib/styles/globals/controls.css

```
@@ -126,14 +139,16 @@
 
 /* ── Composable affordance and size modifiers ───────────────────── */
 
-.lattice-btn--dashed {
-  border-style: dashed;
+.lattice-btn.lattice-btn--dashed {
+  --lattice-btn-ring: transparent;
+  outline: 1px dashed currentColor;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)